### ISSUE TITLE
SKS List Module for Semester, Course, and Student Objects

### ISSUE THREAD

**Issue by robmuh** [skilstak/internal-projects#12]
Need to support the following `sks` commands:

* `sks list` - lists the current context
* `sks list semesters` - lists all the courses and camps for all semesters in summary
* `sks list <semester>` - lists all the courses and camps for specific semester
* `sks list <semester>.<course/camp>` lists all the students, session dates, and staff for course/camp
* `sks list <semester>.<course/camp>.<session>` - lists session details by session id or prefix
* `sks list students` - lists a summary of all students
* `sks list courses` - lists all the courses in datetime aware fashion
* `sks list sessions` - lists all the sessions in datetime aware fashion

**Comment by donovank** [comment]
I thought SKS was dead when main went down.

**Comment by robmuh** [comment]
Nope, `sks` will remain the main interface to all the SkilStak Management
System including the Learning Management subsystem.

*Rob Muhlestein*
@SkilStak <http://twitter.com/skilstak>, Inc.
rob@skilstak.com
704-310-6778
@robmuh <http://twitter.com/robmuh>

**Comment by robmuh** [comment]
Nope, `sks` will remain the primary interface to the SkilStak™ Management System as well as the Learning Management subsystem.
